EKF/ekf.h

```diff
@@ -264,6 +264,24 @@ class Ekf : public EstimatorInterface
 	// set minimum continuous period without GPS fail required to mark a healthy GPS status
 	void set_min_required_gps_health_time(uint32_t time_us) { _min_gps_health_time_us = time_us; }
 
+	// get solution data for the EKF-GSF emergency yaw esitmator
+	void getDataEKFGSF(float *yaw_composite, float yaw[N_MODELS_EKFGSF], float innov_VN[N_MODELS_EKFGSF], float innov_VE[N_MODELS_EKFGSF], float weight[N_MODELS_EKFGSF]) const;
+
+	// gets data which to be logged and used for EKF-GSF development replay
+	// returns false when no data available
+	bool get_algo_test_data(float delAng[3], // delta angle from IMU (rad)
+				float *delAngDt, // delta angle integration time (sec)
+				float delVel[3], // delta velocity from IMU (m/s)
+				float *delVelDt, // delta velocity integration time (sec)
+				float vel[3], // NED velocity measurement (m/s)
+				float *velErr, // 1-sigma velocity measurement uncertainty (m/s)
+				bool *fuse_vel, // true when velociy measurement has been updated
+				float quat[4]); // reference quaternion from EKF
+
+	// request the EKF reset the yaw to the estimate from the internal EKF-GSF filter
+	// argment should be incremented only when a new reset is required
+	void request_ekfgsf_yaw_reset(uint8_t counter);
```

Comment:
The preferred case style for functions is snakeCase

Reply:
Will fix.

Reply:
I've renamed that function to better reflect when it should be used and what it does. See https://github.com/PX4/ecl/pull/751/commits/87b23448807b3f36eb8f053909f683af6be6e9e3